<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scan</class>
 <widget class="QWidget" name="scan">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>468</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ضد‌بد‌افزار حافظ</string>
  </property>
  <property name="windowOpacity">
   <double>75.000000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#scan
	{
	background-image: url(:/image/backForm.png);
	color: rgb(127, 0, 63);
	selection-color: white;
	border: 2px groove gray;
	border-radius: 20px;
	padding: 6px 6px;
	font: 75 8pt &quot;Tahoma&quot;;
	}

#stackedWidget
	{
	border: 0px groove white;
	border-radius: 1px;
	padding: 6px 6px;
	}

a
	{
    padding: inherit;
    list-style-type: circle;
    list-style-image: inherit;
    list-style-position: outside;
    font-family: tahoma;
    background-color: #CCFFCC;
    background-repeat: no-repeat;
    position: fixed;
    visibility: collapse;
	text-decoration:none;
	}

a:hover
	{
    color: #FF00FF;
	}
#start1Button
	{
   position: absolute;
    display: block;
    visibility: visible;
    color: #000080;
    line-height: 3px;
    vertical-align: 5%;
    text-align: leftr;
	background-color: #008000;
	}

#label_3
	{
    font-family: 'B Titr';
    text-decoration: underline;
    color: #800000;
    font-size: large;
    background-color: #C0C0C0;
	}
</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>65</y>
     <width>541</width>
     <height>361</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>20</number>
   </property>
   <property name="midLineWidth">
    <number>22</number>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <widget class="QWidget" name="ScanStackWidget">
    <widget class="QGroupBox" name="rapidGroupBox">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>51</y>
       <width>481</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>8</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(251, 251, 251);
color: rgb(0, 85, 0);
font: 75 8pt &quot;Tahoma&quot;;
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>186</x>
        <y>50</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 10pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>جستجو از حافظه تعیین شده در تنظیمات برنامه</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="automatLabelScan">
      <property name="geometry">
       <rect>
        <x>223</x>
        <y>4</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;B Titr&quot;;</string>
      </property>
      <property name="text">
       <string>برشماری و جستجوی خودکار</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="extendedGroupBox">
     <property name="geometry">
      <rect>
       <x>27</x>
       <y>150</y>
       <width>481</width>
       <height>81</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Tahoma</family>
       <pointsize>8</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 8pt &quot;Tahoma&quot;;
background-color: rgb(251, 251, 251);
color: rgb(0, 85, 0);
</string>
     </property>
     <property name="title">
      <string notr="true"/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>216</x>
        <y>53</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 10pt &quot;Times New Roman&quot;;</string>
      </property>
      <property name="text">
       <string>جستجو از حافظه تعیین شده توسط کاربر </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="manualLabelScan">
      <property name="geometry">
       <rect>
        <x>227</x>
        <y>5</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;B Titr&quot;;</string>
      </property>
      <property name="text">
       <string>برشماری و جستجوی دستی</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="scanLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-3</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="sorce.qrc">:/image/scanLabel.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="PathSelectStackWidget">
    <widget class="QPushButton" name="searchButton">
     <property name="geometry">
      <rect>
       <x>372</x>
       <y>284</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: none;</string>
     </property>
     <property name="text">
      <string>&amp;اضافه</string>
     </property>
     <property name="icon">
      <iconset resource="sorce.qrc">
       <normaloff>:/image/add.png</normaloff>:/image/add.png</iconset>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_PathExtendedSearch">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>58</y>
       <width>391</width>
       <height>221</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">listWidget
{
background-color: rgb(255, 255, 255);
}</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>283</y>
       <width>158</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_13">
      <item row="0" column="1">
       <widget class="QPushButton" name="startScanExtended_Button">
        <property name="text">
         <string>&amp;شروع</string>
        </property>
        <property name="icon">
         <iconset resource="sorce.qrc">
          <normaloff>:/image/accept.png</normaloff>:/image/accept.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="CancelButton">
        <property name="text">
         <string>ا&amp;نصراف</string>
        </property>
        <property name="icon">
         <iconset resource="sorce.qrc">
          <normaloff>:/image/E%20mail.png</normaloff>:/image/E%20mail.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>272</x>
       <y>24</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>مسیرهای انتخاب شده جهت بررسی</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tempPathSelect">
    <widget class="QPushButton" name="p1111111">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ادامه</string>
     </property>
    </widget>
    <widget class="QPushButton" name="p1111111_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>توقف</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="LogStackWidget">
    <widget class="QTextBrowser" name="textLog">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>168</y>
       <width>501</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>69</y>
       <width>501</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#groupBox
{
	background-color: rgb(0, 170, 127);
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QPushButton" name="pushButton_RemoveBetween">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>51</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>حذف کن</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_CurrRemove">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>23</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>گزارش جاری</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_RemoveBetween">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>97</x>
        <y>53</y>
        <width>80</width>
        <height>17</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>بین دو تاریخ</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>189</x>
        <y>15</y>
        <width>3</width>
        <height>61</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_End">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>17</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>316</x>
        <y>19</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>تا</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_Start">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>17</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>1</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="time">
       <time>
        <hour>1</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_Reports">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>50</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>459</x>
        <y>18</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>شروع</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>انتخاب گزارش</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="settingLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="sorce.qrc">:/image/logLabel.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="settingStackWidget"/>
   <widget class="QWidget" name="quarantineWidget">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>-7</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="sorce.qrc">:/image/Quarantine_Label.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="Quarantine_pushButton">
     <property name="geometry">
      <rect>
       <x>406</x>
       <y>308</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>قرنطینه</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Restore_pushButton">
     <property name="geometry">
      <rect>
       <x>326</x>
       <y>308</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>بازیابی</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="header_label">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>0</y>
     <width>791</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="sorce.qrc">:/icons/header.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_6"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3"/>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>573</x>
     <y>65</y>
     <width>151</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QToolButton" name="scanButton">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="logButton">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>49</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="settingButton">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>99</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="Quarantine">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>149</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="aboutUsLabel">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>400</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>B Titr</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>درباره برنامه</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>header_label</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>aboutUsLabel</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sorce.qrc"/>
 </resources>
 <connections/>
</ui>
